changelog/3393.doc.rst

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
You can delete this

Reply:
of course - seems to be a git error on my side :-(
I renamed the file (instead of `git mv`), committed  and pushed.

Reply:
That is perfectly fine.